test/modules/sungeun/init/printModuleInitOrder.good

```diff
@@ -42,6 +42,8 @@ Initializing Modules:
          ChapelHashtable
         ExternalArray
         RangeChunk
+        Search
+         HideDeprecatedReexport
```

Comment:
Why is `Search` being printed when it wasn't before? Did adding the submodule the module initialization?

Reply:
Search is used by DefaultSparse.  I think the reason it's turning up here is something to do with how the shortest path to the symbol is calculated.  It's pretty annoying, but the explicit deprecated symbol implementation also led to it turning up.  So I think we're just getting access to Sort in a longer chain of use statements than is reasonable somewhere else, but tracking that down seems like a lower priority than many other fixes (much fun as it would be to have our module use chain structure simplified and cleaner)

Reply:
It's also possible that our changes to visible symbol determination in the last few years had bugs that are only apparently in more complicated use chains